src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -192,7 +273,7 @@ class JesBackend extends Backend with LazyLogging {
   private def runWithJes(backendCall: BackendCall, command: String, jesInputs: Seq[JesParameter], jesOutputs: Seq[JesParameter]): ExecutionResult = {
     val tag = makeTag(backendCall)
     // FIXME: Ignore all the errors!
```

Comment:
I'd love to know what this fixme was about - if anyone has an idea please speak up :)